### PR TITLE
chore: update CHANGELOG for v0.26.0 release

### PR DESCRIPTION
**Pull request by NagyZoltanPeter**
CHANGELOG for next, v0.26.0 release!

@jm-clius: Please make your point on decomissioning Filter v1 and its API in the next release. Thank you.

CC: @chair28980

### CODE REVIEW

**Comment by gabrielmer** [comment]
LGTM! Thanks so much!

The markdown here looks a bit weird I think


Screenshot 2024-03-07 at 10 09 16

Maybe in the `Note` part we should have the 3 bullet points in the same format

**Review by gabrielmer — APPROVED** [review]

**Review by rymnc — APPROVED** [review]
LGTM, I'd like to get a couple prs merged before the release though, when will you be creating the release?

**Comment by NagyZoltanPeter** [comment]
> LGTM! Thanks so much!
> 
> The markdown here looks a bit weird I think
> 
> Screenshot 2024-03-07 at 10 09 16
> Maybe in the `Note` part we should have the 3 bullet points in the same format

Thank you. It is fixed now!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,84 @@
+## v0.26.0 (2024-03-07)
+
+> **Note:**
+> - JSON-RPC API has been removed completely. Instead we recommend you to utilize REST API endpoints that have same and extended functionality.
+> Please have a look at Waku's REST-API reference: https://waku-org.github.io/waku-rest-api
+> - Support for Cross-Origin-Resource-Sharing (CORS headers) is added for our REST-API services. This allows you to access our REST-API from a different domain from a browser.
```
Comment (Ivansete-status) [inline CHANGELOG.md:6]: nitpick, reworking a little bit. Maybe that suggestion is not correct but the original one sounds a bit unclear.
```suggestion
> - Support for Cross-Origin-Resource-Sharing (CORS headers) is added for our REST-API services. This allows you to access our REST-API from a browser.
```

**Review by Ivansete-status — APPROVED** [review]
LGTM! Thanks!
Just added minor nitpick comment:)

**Review by jm-clius — APPROVED** [review]
LGTM!

**Review by SionoiS — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (NagyZoltanPeter) [reply CHANGELOG.md:6]: Accept!

**Pull request merged**